### PR TITLE
Debian based

### PR DESCRIPTION
**Pull request by tobiasge**
Related Issue: #539, #654

## New Behavior
-  Images are based on Debian Bullseye

## Contrast to Current Behavior
- Images based on Alpine 3.14

## Discussion: Benefits and Drawbacks
- See in  this PR #604
- This is the first step to enabled builds for ARM64

## Changes to the Wiki
- All pages need to be checked for Alpine specific commands

## Proposed Release Note Entry
- Changed images to be based on Debian

## Double Check
* [x] I have read the comments and followed the PR template.
* [x] I have explained my PR according to the information in the comments.
* [x] My PR targets the `develop` branch.

### CODE REVIEW

File: requirements-container.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ django-auth-ldap==4.1.0
 django-storages[azure,boto3,dropbox,google,libcloud,sftp]==1.12.3
 google-crc32c==1.3.0
```
Comment (cimnine) [inline requirements-container.txt]: Do we still need it explicitly in our `requirements.txt`?

File: build.sh [hunk 2]
```diff
@@ -170,7 +170,7 @@ fi
 # Determining the value for DOCKER_FROM
 ###
 if [ -z "$DOCKER_FROM" ]; then
-  DOCKER_FROM="alpine:3.14"
+  DOCKER_FROM="debian:bullseye-slim"
```
Comment (cimnine) [inline build.sh]: IMO using the version number (here 11-slim) is preferred as not all are that familiar with the release names in Debian. But at least a number is monotonic.

**Review by cimnine — APPROVED** [review]

File: requirements-container.txt (hunk 1 above)
Comment (tobiasge) [reply requirements-container.txt]: I don't think so. Will remove it.

File: build.sh (hunk 2 above)
Comment (tobiasge) [reply build.sh]: Ok, i will change it to the number.

**Pull request merged**